pkgs/development/python-modules/asyncio-throttle/default.nix

```
@@ -0,0 +1,38 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, pytestCheckHook
+, pytest-asyncio
+}:
+
+buildPythonPackage rec {
+  pname = "asyncio-throttle";
+  version = "1.0.1";
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "hallazzang";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0raqnrnp42cn1c7whbm7ajbgaczx33k6hbxsj30nh998pqxhh4sj";
+  };
+
+  checkInputs = [
+    pytest-asyncio
+    #pytest-trio
+    #pytest-tornasync
+    #pytest-twisted
+    pytestCheckHook
+    #twisted
+  ];
```

remove if not needed
```suggestion
  checkInputs = [
    pytest-asyncio
    pytestCheckHook
  ];
```

---

Leftovers, had to choose one runtime to run the tests with. Removed!